static/editjs.js

```diff
@@ -0,0 +1,16 @@
+$(document).ready(function(){
+    $("button#tweetbutton").click(function(){
+        tweet_entry();
+    });
+    $("button#editbutton").click(function(){
+      add_entry_to_db();
+    });
+});
+
+function add_entry_to_db() {
+    $.post("", {'title' : $("[name=title]").val(), 'text' : $("[name=text]").val()});
+}
```

Comment:
What happens after you've called `$.post` here?  Do you just sit on the same page, looking at the same entry?